sql/electives.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE electives (
```

Comment:
Please also add create and use statements for the DB. This way the script can be copy-pasted into any other machine and the code will work out of the box. Reference: https://github.com/aprazova/WEB2020-php/pull/9/files